src/ontology/cob.Makefile

```diff
@@ -54,36 +54,43 @@ cob.tsv: cob.owl
 # -- MAIN RELEASE PRODUCTS --
 ########################################
 
-$(ONT)-edit.owl: $(TMPDIR)/cob-root.tsv $(COMPONENTSDIR)/obsolete.tsv
+# Include all terms
+cob-edit.owl: $(TMPDIR)/cob-root.tsv $(COMPONENTSDIR)/obsolete.tsv patch-labels.ru patch-definitions.ru
 	$(ROBOT) template \
 	$(foreach X,$(filter %.tsv,$^),--template $(X)) \
+	query \
+	$(foreach X,$(filter %.ru,$^),--update $(X)) \
```

Comment:
What happens if in upstream ontologies changes its mind and adds @en? Probably the pipeline will fail und someone needs to know that they have to edit the sparql query, right?

Reply:
Yes, that's the design: be explicit. @sebastianduesing and I will document all this.